src/main/java/ddingdong/ddingdongBE/common/exception/CustomExceptionHandler.java

```diff
@@ -125,4 +149,22 @@ private String createLogConnectionInfo(HttpServletRequest request) {
     private void loggingApplicationError(String applicationLog) {
         log.warn("errorLog = {}", applicationLog);
     }
+
+    public String getRequestBody(HttpServletRequest request) {
+        try {
+            InputStream inputStream = request.getInputStream();
+            BufferedReader reader = new BufferedReader(
+                new InputStreamReader(inputStream, StandardCharsets.UTF_8));
+            StringBuilder stringBuilder = new StringBuilder();
+            String line;
+
+            while ((line = reader.readLine()) != null) {
+                stringBuilder.append(line);
```

Comment:
```suggestion
                stringBuilder.append(line).append("\n");
```
혹시 줄바꿈을 해줄 필요는 없을까요??